docs/guide.md

```diff
@@ -300,6 +300,22 @@ func injectFooBar() FooBar {
 
 And similarly if the injector needed a `*FooBar`.
 
+Sometimes it is useful to restrict certain fields to be accidentally filled by
+the injector. You can use a struct tag, `wire:"-"` to mark such fields like in
+the following example:
+
+```go
+type Foo struct {
+    mu sync.Mutex `wire:"-"`
+    Bar Bar
+}
+```
+
+When you provide the `Foo` type using `wire.Struct(new(Foo))`, wire will
+automatically omit the `mu` field. On the other hand, when you specify the
```

Comment:
"On the other hand" implies viewing the same situation from a different perspective, whereas this is adding information about a new situation.

```suggestion
automatically omit the `mu` field. Additionally, when you specify the
```

Reply:
Done.